hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/AbstractJobProxy.java

```diff
@@ -63,21 +63,19 @@
     private final AtomicBoolean cancelledJob = new AtomicBoolean();
     private final ExecutionCallback<Void> joinJobCallback = new JoinJobCallback();
 
-    private final Supplier<JobConfig> jobConfigSup;
+    private volatile JobConfig jobConfig;
     private final Supplier<Long> submissionTimeSup = memoizeConcurrent(this::doGetJobSubmissionTime);
 
     AbstractJobProxy(T container, long jobId) {
         this.jobId = jobId;
-        this.jobConfigSup = memoizeConcurrent(this::doGetJobConfig);
```

Comment:
what was this inlined?

Reply:
because i needed to get the value without triggering a lazy load. It's needed in case of various errors where the could fail too.